decorators/5.py

```diff
@@ -0,0 +1,26 @@
+import time
+
+
+def timethis(func):
+    def wrapper(*args, **kwargs):
+        start_time = time.time()
+
+        func(*args, **kwargs)
+
+        end_time = time.time()
```

Comment:
ciekawostka - można użyć `perf_counter()` dla większej dokładności
https://medium.com/bitgrit-data-science-publication/4-ways-to-time-python-functions-bb2e4fbe4121